### PR TITLE
Update team-meetings.md

### PR DESCRIPTION
**Pull request by kniz-raft**
### This pull request changes...

*Updated attendees*

## FUNCTIONALITY
*Describe the functionality*
*Addresses Issue #*

## TO TEST

*List the steps to test the PR*

### This pull request is ready to merge when...

- [ ] Meets acceptance criteria for the issue 
- [ ] Code is reviewed by someone other than the original author
- [ ] Code is tested 
- [ ] Experience is approved by UX  
- [ ] Meets [a11y checklist](https://github.com/HHS/TANF-app/blob/main/docs/a11y/how-18f-will-test-a11y.md)
- [ ] Meets [Raft's Manual QASP Checklist](https://teams.microsoft.com/l/file/B5285301-13CA-46FD-A723-AFFD578C224A?tenantId=d58addea-5053-4a80-8499-ba4d944910df&fileType=docx&objectUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA%2FShared%20Documents%2FGeneral%2FManual%20Testing%20Practices_Issue173.docx&baseUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA&serviceName=teams&threadId=19:f769bbcb029f4f02b55ae7fad90e310d@thread.skype&groupId=41f194a6-c1d3-4680-933e-c8ee7d17e287):lock: (only applicable for PR to main HHS)
- [ ] Documentation is updated 
  - OpenAPI  
  - Readme
  - Entity Relationship Diagram (ERD)

### CODE REVIEW

**Comment by kniz-raft** [comment]
@lfrohlich

File: docs/How-We-Work/team-charter/team-meetings.md [hunk 1]
```diff
@@ -22,21 +22,21 @@
 
 ## Backlog Refinement 
 - Goal: To identify goals for next sprint and create issues to be discussed in Sprint Planning 
-- Attendees: Lauren, Alex, Shubhi, Carl, Matthew. On alternate weeks, Ryan, Penyin, Tijan.
-- Facilitators: Lauren + Shubhi
+- Attendees: Lauren, Alex, Shubhi, Carl, Brian. On alternate weeks, Ryan, Penyin, Tijan.
```
Comment (lfrohlich) [inline docs/How-We-Work/team-charter/team-meetings.md:25]:
```suggestion
- Attendees: Lauren, Alex S., Shubhi, Carl, Brian, Jennifer, Kati. On alternate weeks, Penyin, Tijan, and Patrick.
```

File: docs/How-We-Work/team-charter/team-meetings.md [hunk 2]
```diff
@@ -22,21 +22,21 @@
...
 - Attendees: Core team 
-- Facilitators: Lauren + Shubhi
+- Facilitators: Lauren + Kati/Shubhi
 - When: Every other Wednesday 3:30-5:00pm EST 
 - Format: Product owner should come with priority cards prepared and ready to discuss with the rest of the team. Work in progress should be assessed and the team should decide if work should continue into the new sprint. Team will flesh out issues, acceptence criteria, define done, and right-size the amount of work for the sprint.  
 
 ## Sprint Demo Prep  
 - Goal: To plan what will be demoed in Sprint Demo 
-- Attendees: Lauren, Alex, Shubhi, Carl, Matthew 
+- Attendees: Lauren, Alex, Shubhi, Carl, Kati
```
Comment (lfrohlich) [inline docs/How-We-Work/team-charter/team-meetings.md:39]:
```suggestion
- Attendees: Lauren, Alex, Shubhi, Carl, Kati, Jennifer 
```

**Review by lfrohlich — APPROVED** [review]
Looks good

**Pull request merged**